doc/abrt-oops.conf.txt

```diff
@@ -5,6 +5,10 @@ NAME
 ----
 abrt-oops.conf - Configuration file for ABRT's Kernel Oops extractor
 
+SYNOPSIS
+--------
+/etc/abrt/plugins/vmcore.conf
```

Comment:
This doesn’t look right.

Reply:
Putting the path into the synopsis? I've sometimes seen that (for instance `resolv.conf`), although I don't have a strong opinion on how "correct" it is. Do you think it would be better to only keep it in the files section at the bottom?

Reply:
I mean the actual path. It’s not for the same file.